lib/file-finder.js

```diff
@@ -113,20 +114,23 @@ class FileFinder {
                         // Add the file path to the cache of each directory searched.
                         for (let j = 0; j < searched; j++) {
                             cache[dirs[j]].push(filePath);
+                            yield [filePath];
```

Comment:
Is it a problem that an empty object is returned? I would assume that this just means a config file was empty (although I might be misunderstanding the issue).